src/Command/App.php

```diff
@@ -66,14 +66,14 @@ protected function configure(): void
      * @throws InvalidArgumentException
      * @throws RuntimeException
      */
-    protected function execute(InputInterface $input, OutputInterface $symfonyOutput): int
+    protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $directory    = $input->getArgument('directory');
         $outputOption = $input->getOption('output');
-        $output       = new Stdout($symfonyOutput);
+        $stdout       = new Stdout($output);
 
         if ($outputOption === 'checkstyle') {
-            $output = new Checkstyle($symfonyOutput);
+            $stdout = new Checkstyle($output);
```

Comment:
Since it can be `Checkstyle` or `Stdout` I think $output was better.

Reply:
Maybe use the interface name and go with `$outputFormatter`.
```suggestion
            $outputFormatter = new Checkstyle($output);
```